.github/workflows/deploy-app.yml

```diff
@@ -52,6 +52,6 @@ jobs:
           become mismatch-finder chmod -R g+rwx ~tools.mismatch-finder/mismatch-finder-repo
           # Change group of ~/mismatch-finder-repo (including symlinks) to tools.mismatch-finder
           chgrp --no-dereference -R tools.mismatch-finder ~/mismatch-finder-repo
-          rsync -rlgD --delete --delay-updates --exclude '.nfs*' --exclude mismatch-finder-repo/.env --exclude mismatch-finder-repo/storage/app/ ~/mismatch-finder-repo ~tools.mismatch-finder/
+          rsync -rlgD --delete --delay-updates --exclude '.nfs*' --exclude .env --exclude storage/app/ ~/mismatch-finder-repo ~tools.mismatch-finder/
```

Comment:
this does the same thing, but is much shorter (esp in the staging version)

Reply:
thank you, it is!